test/e2e-go/cli/goal/expect/corsTest.exp

```diff
@@ -18,6 +18,9 @@ if { [catch {
     set NETWORK_TEMPLATE "$TEST_DATA_DIR/nettemplates/TwoNodes50Each.json"
 
     # Create network
+    ::AlgorandGoal::CreateNetwork $NETWORK_NAME $NETWORK_TEMPLATE $TEST_ALGO_DIR $TEST_ROOT_DIR
```

Comment:
Did you separate the creation of the network from the starting of the network, to make it easier to start and restart the same network?  Or was there another reason?

Reply:
I explained that in the PR title. I broke that functionality into the goal-oriented functions because it's the only way to create the network first, modify some of it's parameters, and start the nodes on the network in a specific order.